Package.swift

```diff
@@ -23,11 +26,18 @@ let package = Package(
     targets: [
         .target(
             name: "Layout"),
+        .target(
+            name: "SnapshotTestingExtensions",
+            swiftSettings: .swiftSettings,
+            plugins: [
+                .plugin(name: SwiftLint.plugin),
+            ]),
         .testTarget(
             name: "LayoutTests",
             dependencies: [
                 "Layout",
                 "Nimble",
+                "SnapshotTestingExtensions",
```

Comment:
Place immediately following `Layout` so it is together with the other target in this package.
```suggestion
                "Layout",
                "SnapshotTestingExtensions",
                "Nimble",
```